samza-core/src/main/java/org/apache/samza/container/grouper/task/GroupByContainerCount.java

```diff
@@ -225,10 +225,11 @@ private void validateTasks(Set<TaskModel> tasks) {
    * @param taskNamesToAssign     the list of tasks to assign to the containers.
    * @param containers            the containers (as {@link TaskGroup}) to which the tasks will be assigned.
    */
+  // TODO: Change logic from using int arrays to ordered array list
```

Comment:
Oh.. That is a really old comment when I had not fully read through the implementation of the class. Let me fix the comment. Thanks!